app/src/main/res/layout/fragment_skill_listing.xml

```diff
@@ -12,7 +12,8 @@
         <android.support.v7.widget.RecyclerView
             android:id="@+id/skillMetrics"
             android:layout_width="match_parent"
-            android:layout_height="match_parent" />
+            android:layout_height="match_parent"
+            android:padding="8dp"/>
```

Comment:
@chanmol1999 I feel you could have added a few more enhancements if any while keeping the UI consistent with that on CMS and iOS app as well.
Also, please do not use hardcoded dimensions. :)

Reply:
Okay I'll try to add few more enhancements @arundhati24